docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -125,6 +125,51 @@ include::docker/initialize.adoc[]
 endif::[]
 ====
 
+== Receive SNMP Traps/Informs
+
+OpenNMS {page-component-title} core lets you receive and process SNMP Traps/Informs out of the box.
+The OpenNMS {page-component-title} core services run as an unprivileged user and can't bind on port numbers < 1024 without escalated privileges.
+For this reason, the default port for the SNMP Trap/Inform listener is set to port number 10162/udp instead of the IANA registered port number 162/udp.
+The following example shows how to configure the local firewall daemon to forward port 162/udp to 10162/udp.
+
+TIP: If you need SNMP Trap listener on port 162/udp directly, you can add the Linux `CAP_NET_BIND_SERVICE` capability to the Java binary using `setcap`.
+     Be aware: this method allows any Java program executed on your system to bind to privileged ports < 1024.
+
+[{tabs}]
+====
+CentOS/RHEL 7/8::
++
+--
+include::centos-rhel/firewall-core.adoc[]
+--
+
+ifeval::["{page-component-title}" == "Horizon"]
+Debian/Ubuntu::
++
+--
+include::debian-ubuntu/firewall-core.adoc[]
+--
+endif::[]
+====
+
+You can verify your firewall and portforwarding configuration by sending a SNMP trap from a remote system to your OpenNMS {page-component-title} core instance with the following command.
+
+[source, console]
+----
+snmptrap -v 2c -c public opennms-core-host '' 1.3.6.1.4.1.2021.991.17 .1.3.6.1.2.1.1.6.0 s "Milky Way"<1><2>
+----
+<1> OpenNMS has by default the community `public` configured, if you have modified it in OpenNMS you have to use your own community name here.
+<2> Replace `opennms-core-host` with the IP or FQDN of your OpenNMS {page-component-title} core instance.
+
+On RHEL/CentOS the `snmptrap` command line tool is part of the `net-snmp-utils`. 
+When you run on Debian/Ubuntu you have to install the `snmp-utils` package.
+
+Your configuration works as expected when you see an SNMP trap event in the web user interface.
+
+. Log in to the web user interface
+. Click menu:Status[Events > All Events]
+. Verify if you have received an `uei.opennms.org/generic/traps/EnterpriseDefault` event from your test host
```

Comment:
```suggestion
. Verify you received a `uei.opennms.org/generic/traps/EnterpriseDefault` event from your test host.
```
I have to say, the "a/an" rule before a vowel is very strange in English, since it's based not on the vowel, but on whether the sound of the vowel is vowel-like or consonant like. Which assumes that when one reads, one is making the sounds in one's head. :-/